Get-NetView.psm1

```diff
@@ -1131,6 +1131,195 @@ function MellanoxDetail {
     MellanoxDetailPerNic -NicName $NicName -OutDir $dir
 } # MellanoxDetail()
 
+function MarvellDetail{
```

Comment:
If there are any Marvell specific counters you would like to collect, please add an appropriate filter to `$pathFilters` in function Counters

Reply:
Added Marvell counters. Thanks for highlighting it.